client/blocks/subscription-length-picker/test/index.js

```diff
@@ -0,0 +1,100 @@
+/** @format */
+
+jest.mock( 'lib/abtest', () => ( {
+	abtest: () => '',
+} ) );
+
+const translate = x => x;
+jest.mock( 'i18n-calypso', () => ( {
```

Comment:
Instead of mocking, we can import what we need from i18n directly ;) We can pass through `translate` as a prop. eg. `identity` works too if our strings are not complex.

Reply:
Sounds good, I am removing that - this was introduced so we could look for `Component` instead of `Localized(Component)`, but in the meantime I found another way to achieve that without mocking i18n-calypso.

Reply:
For component testing we normally test with the unwrapped class. Mocking i18n isn't necessary.